examples/resources/gateway_config_store.tf

```diff
@@ -0,0 +1,15 @@
+resource "konnect_gateway_config_store" "my_configstore" {
+  name  = "Config Store"
+  force = "true"
+
+  control_plane_id = konnect_gateway_control_plane.tfdemo.id
+}
+
+resource "konnect_gateway_vault" "my_vault" {
+  name   = "konnect"
```

Comment:
Bit of a nitpick, but on the UI, the name field here translates to the `Vault Configuration` property, correct? and prefix (here "my-konnect-vault") is shown in the heading. This might be a tad bit confusing to users 😅 
Although, since we want to be consistent with the API, this is probably fine.

Reply:
Terraform is aligned to the API. It is confusing, but it is also what’s in production